dask/dask-schema.yaml

```diff
@@ -135,6 +135,17 @@ properties:
               The graph growth factor above which task-based shuffling introduces
               an intermediate step.
 
+      shuffle:
+        type: object
+        properties:
+
+          chunksize-tolerance:
+            type: number
+            description: |
+              Tolerance for the shuffle algorithm when creating output chunks.
+              Default is 1.25. This means that the shuffle algorithm can exceed
+              the average input chunk size along this dimension by 25%.
```

Comment:
Do we also impose a lower limit? When thinking about tolerances, I think about limits in both directions. If it's only an upper limit, it might make sense to make this explicit in the naming.

Reply:
only upper,